app/App.js

```diff
@@ -34,17 +37,26 @@ class App extends React.Component<Props, State> {
 
   healthCheckInterval: IntervalID;
 
+  updateCheckInterval: IntervalID;
+
   checkConnectionTimer: TimeoutID;
 
   state = {
-    error: null
+    error: null,
+    isUpdateDownloaded: false
   };
 
   render() {
-    const { error } = this.state;
+    const { error, isUpdateDownloaded } = this.state;
+
     if (error) {
       return <ErrorHandlerModal componentStack={''} explanationText="Retry failed action or refresh page" error={error} onRefresh={nodeService.hardRefresh} />;
     }
+
+    if (isUpdateDownloaded) {
+      return <UpdaterModal onCloseModal={this.closeUpdateModal} />;
```

Comment:
move this down inside <> at the same level as GlobalStyle and make it a ternary rendering